components/camel-cxf/camel-cxf-rest/src/main/java/org/apache/camel/component/cxf/jaxrs/CxfRsProducer.java

```diff
@@ -24,11 +24,7 @@
 import java.net.CookieStore;
 import java.net.HttpCookie;
 import java.net.URLDecoder;
-import java.util.Collection;
-import java.util.HashMap;
-import java.util.LinkedHashMap;
-import java.util.List;
-import java.util.Map;
+import java.util.*;
```

Comment:
Please avoid wildcard import

Reply:
yeah, the IDE thinks he's smart, I'll review them and update the PR